.github/workflows/as-docker-build.yml

```diff
@@ -15,25 +15,9 @@ on:
   create:
 
 jobs:
-  basic_ci:
+  check_as_image_build:
     if: github.event_name == 'pull_request' || github.event_name == 'push'
-    name: Check
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-
-    steps:
-    - name: Code checkout
-      uses: actions/checkout@v4
-    
-    - name: Build gRPC AS Container Image
-      run: |
-        DOCKER_BUILDKIT=1 docker build -t attestation-service:latest . -f attestation-service/docker/as-grpc/Dockerfile
-    
-    - name: Build RESTful AS Container Image
-      run: |
-        DOCKER_BUILDKIT=1 docker build -t attestation-service:latest . -f attestation-service/docker/as-restful/Dockerfile
-
-    - name: Build RVPS Container Image
-      run: |
-        Docker_BUILDKIT=1 docker build -t rvps:latest . -f rvps/docker/Dockerfile
\ No newline at end of file
+    uses: ./.github/workflows/build-as-image.yml
+    with:
+      build_option:
```

Comment:
Can we make this optional on the build-as-image workflow, so we don't have to specify an empty build_option?

Reply:
Yes, we could make it optional. I will update it.